eng/.docsettings.yml

```
@@ -20,6 +20,7 @@ omitted_paths:
   - sdk/resourcemanager/*
   - src/*
   - tools/*
+  - sdk/batch/Microsoft.Azure.Batch/tools/*
```

```suggestion
  - sdk/**/tools/*
```

Not sure if we can use splat patterns, but it may make sense to see if we can apply this to all embedded `tools` folders under the SDK slug.

---

tested in https://github.com/Azure/azure-sdk-for-net/pull/45195, it works, thanks!